### PR TITLE
Drop dynamic severity classes

### PR DESCRIPTION
**Pull request by janowagner**
**What**:

The support for multiple severity class ranges is removed.
There is now only one classification scheme (then one previously
called "nist"). Actually there was only one other classification scheme called
"pci-dss". It wasn't used or needed.

For the user, this essentially means that in "My Settings", the option
to select a severity classification scheme is not there anymore and that
the default is now applied mandatory.

This PR requires: https://github.com/greenbone/gvmd/pull/1288

**Why**:

Removing the overhead about a dynamic handling where actually only static
class ranges are used simplifies code.
  

**How**:

Basically all aspects are removed, on gsa side, and on gvmd side.
While gvmd appears pretty clean now, in gsa there are some special
handlings with an old range style due to compatibility with older dashboard elements.

**Checklist**:

- [ ] Tests N/A
- [x] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

**Comment by bjoernricks** [comment]
I am a bit surprised that no tests had to be adjusted. @janowagner did you check all charts if they still work?

**Comment by janowagner** [comment]
> I am a bit surprised that no tests had to be adjusted. @janowagner did you check all charts if they still work?

Indeed this code was no covered by tests. I checked all charts manually and all looked fine.

**Review by bjoernricks — APPROVED** [review]
